stache.yml

```diff
@@ -153,6 +155,11 @@ github_token:
 github_org: blackbaud
 github_repo:
 github_branch: master
+vsts_protocol: https://
+vsts_base: blackbaud.visualstudio.com
+vsts_token:
+vsts_project:
+vsts_branch: master
```

Comment:
Even though I know the end-user needs to specify it, for consistency of known config params should we add `vsts_repo` here?